### PR TITLE
Create performance.yml

### PR DESCRIPTION
**Pull request by hbalasu1**
Create a new performance issue template for customers to report performance-related issues.

### Details:
 - New Performance issue template 

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/performance.yml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      label: OpenVINO Version 
+      description: OpenVINO version, branch, or tag in OpenVINO GitHub
+      placeholder: 2021.4.0 LTS / Master Branch / tag 2022.3.0
+    validations:
+      required: false
+  - type: input
+    id: target_platform
+    attributes:
+      label: Target Platform
+      description: The platform that the OpenVINO source is built on
```
Comment (wangleis) [inline .github/ISSUE_TEMPLATE/performance.yml:22]: Please add `log of "lscpu" and "lscpu -e"` for Linux

File: .github/ISSUE_TEMPLATE/performance.yml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+      placeholder: 11th Gen Intel Core i7-1185G7
+    validations:
+      required: false
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating System
+      description: What OS are you using?
+      options:
```
Comment (wangleis) [inline .github/ISSUE_TEMPLATE/performance.yml:31]: Please add virtual machine information like wsl2 on Windows

File: .github/ISSUE_TEMPLATE/performance.yml [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  - type: dropdown
+    id: device_use
+    attributes:
+      label: Device used for inference
+      description: What hardware are you using for inference?
+      options:
+        - CPU
+        - iGPU
+        - dGPU
+        - VPU
```
Comment (wangleis) [inline .github/ISSUE_TEMPLATE/performance.yml:53]: Please use NPU which is new name of VPU

File: .github/ISSUE_TEMPLATE/performance.yml [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+    attributes:
+      label: Quantized model?
+      description: Is your model quantized with model optimized?
+      options:
+        - 'Yes'
+        - 'No'
+    validations:
+      required: true
+  - type: textarea
+    id: performance_description
```
Comment (wangleis) [inline .github/ISSUE_TEMPLATE/performance.yml:109]: Please add application information, benchmark_app or customer application. 

If this is issue on customer application. could this issue be reproduced on benchmark_app?

**Comment by devangaggarwal** [comment]
@hbalasu1 please address comments above.

File: .github/ISSUE_TEMPLATE/performance.yml (hunk 4 above)
Comment (hbalasu1) [reply .github/ISSUE_TEMPLATE/performance.yml:109]: I put it as description below: 
 Please provide a detailed description of what happened
 let us know the application you use for performance tests
 e.g., hello_classification.py / benchmark_app / own test script
 if you are using your test script, could this reproduce using benchmark_app?

File: .github/ISSUE_TEMPLATE/performance.yml (hunk 3 above)
Comment (hbalasu1) [reply .github/ISSUE_TEMPLATE/performance.yml:53]: Done admend

File: .github/ISSUE_TEMPLATE/performance.yml (hunk 2 above)
Comment (hbalasu1) [reply .github/ISSUE_TEMPLATE/performance.yml:31]: add another line for "WSL2 for windows"

File: .github/ISSUE_TEMPLATE/performance.yml (hunk 1 above)
Comment (hbalasu1) [reply .github/ISSUE_TEMPLATE/performance.yml:22]: Added in the description, so that customer can run the command and copy-paste.

**Comment by hbalasu1** [comment]
Hi @wangleis @devangaggarwal 
I have amended it, and please review it again. 

Thank you

**Comment by hbalasu1** [comment]
Hi @ilya-lavrenov @ilyachur could you help to review this and merge if possible? 
Thank you

**Pull request merged**